andela/coding-geeks-ah-frontend
# #169261050 add proper navbar functionalities

### What does this PR do?
- Add navbar functionalities
- Responsiveness of navbar
### How should this be manually tested?
- Clone this branch
- Run the server by `npm start`
- Check how nav looks like.
### What are the relevant pivotal tracker stories?
- [Add navbar](https://www.pivotaltracker.com/story/show/169261050)
### Use these credentials to login
- email: carlos@gmail.com
- password: User1234
